.github/workflows/support.yml

```
@@ -0,0 +1,47 @@
+name: Support issues
+on:
+  issues:
+    types:
+      - labeled
+
+permissions:
+  issues: write
+
+jobs:
+  support-enviroment:
+    if: github.event.label.name == 'support:environment'
+    uses: ./.github/workflows/close-as-support.yml
+    secrets: inherit
+    with:
+      number: ${{ github.event.issue.number }}
+      reason: >
+        A GitHub Desktop team member has looked at this issue and believes it is
+        not the result of a bug in the application itself but rather an issue
+        with your local environment that would need troubleshooting to be
+        resolved.
+  support-windows-gpu:
+    if: github.event.label.name == 'support:windows-gpu'
+    uses: ./.github/workflows/close-as-support.yml
+    secrets: inherit
+    with:
+      number: ${{ github.event.issue.number }}
+      reason: >
+        A GitHub Desktop team member has looked at this issue and believes it is
+        not the result of a bug in the application itself but rather an issue
+        with your local environment. More specifically it appears this is likely
+        the result of your graphics card or graphics driver and GitHub Desktop
+        not playing nice with each other.
```

```suggestion
        
        It appears this is likely the result of your graphics card or graphics driver.
```

Just thinking that since this is already a really long comment in addition to the close-as-support  comment, it may be better to be more succinct?

---

Lowercase `it`?